lib/value/src/value/serde.rs

```diff
@@ -190,6 +190,28 @@ impl From<serde_json::Value> for Value {
     }
 }
 
+impl From<&serde_json::Value> for Value {
```

Comment:
For this `From` implementation, could we just clone the `json_value` and call the existing `From` implementation for `serde_json::Value`?